python_src/src/lamp_py/performance_manager/l1_rt_trips.py

```diff
@@ -543,12 +541,54 @@ def update_stop_sequence(db_manager: DatabaseManager) -> None:
                 == StaticTrips.static_version_key,
             ),
         )
+        .where(
+            StaticRoutePatterns.static_version_key
+            == sa.func.any(
+                sa.func.array(
+                    sa.select(TempEventCompare.static_version_key)
+                    .distinct()
+                    .scalar_subquery()
+                )
+            ),
+            sa.or_(
+                StaticRoutePatterns.route_pattern_typicality == 1,
+                StaticRoutePatterns.route_pattern_typicality == 5,
+            ),
+        )
+        .order_by(
+            sa.func.coalesce(
+                StaticTrips.branch_route_id, StaticTrips.trunk_route_id
+            ),
+            StaticRoutePatterns.direction_id,
+            StaticRoutePatterns.static_version_key,
+            StaticRoutePatterns.route_pattern_typicality.desc(),
+        )
+        .subquery("canon_trips")
+    )
+    partition_by = (
```

Comment:
i think this could be renamed `stop_sequence_partition` to be a bit more clear?

could also just add it to the `static_cannon` subquery now that its not as large.